### DISCUSSION TITLE
Does Grafana have a way to configure the global access path?

### DISCUSSION THREAD

**Discussion by admin123-cloud** [grafana/grafana#60792] [General]
hi everyone,
as the title, i need  a configuration  to set Grafana page can access by global access address. 

[scene]
i want to manager several grafana cluster , and i want to show the page of grafana in the same place, so  i  use  Nginx to Reverse proxy  this grafana pages. 

but i can't find usable configs ,to change cluster access path   in grafana.ini.  like root_url 、cdn_path
1.root_url can only  proxy one grafana cluster, because  all the request is Relative path.
2.cdn_path can make  some  static file(js 、css ..ect)  to global path , but api query path still use origin host.
like: /api/ds/query

grafana cluster:
http://192.168.1.100:3000/?orgId=1
http://192.168.2.200:3000/?orgId=1
http://192.168.3.300:3000/?orgId=1

 nginx -> grafana ，i wish when  i access anyone of grafana，The subrequest attached to the home page can directly follow there Original domain.

anyone have any experience about this , teach me，please!!
thanks，sincerely!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
